src/containers/guest-home-page/how-it-works/HowItWorks.jsx

```diff
@@ -40,21 +40,21 @@ const HowItWorks = () => {
           {t('guestHomePage.howItWorks.title')}
         </Typography>
         <AppContentSwitcher
-          active={isStudent}
+          active={isTutor}
           onChange={onChange}
           styles={styles.switch}
           switchOptions={switchOptions}
           typographyVariant={'h6'}
         />
         <CardsWithButton
-          array={isStudent ? tutorCardBoxArray : studentCardBoxArray}
+          array={isTutor ? tutorCardBoxArray : studentCardBoxArray}
           btnText={
-            isStudent
+            !isTutor
               ? t('guestHomePage.whatCanYouDo.learn.actionLabel')
               : t('guestHomePage.whatCanYouDo.teach.actionLabel')
           }
-          isStudent={isStudent}
-          role={isStudent ? tutor : student}
+          isStudent={isTutor}
```

Comment:
This is a strange naming, why are we using properties called `isStudent` with a value of `isTutor`?

Reply:
The naming was wrong from the very beginning. We fixed it in the `HowItWorks` component. Should we fix it in the `CardsWithButton` component in the scope of this PR?

Reply:
In 'How It Works', the property of the CardWithButton component has been corrected.